example-utils/hello-world-protos/build.rs

```diff
@@ -0,0 +1,117 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+// use prost_build::Config;
+use protobuf_codegen::Customize;
+use std::env;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+fn main() -> std::io::Result<()> {
+    // use vendored protoc instead of relying on user provided protobuf installation
+    env::set_var("PROTOC", protoc_bin_vendored::protoc_bin_path().unwrap());
+
+    // if let Err(err) = get_and_build_protos(
+    if let Err(err) = get_and_build_protos(
+        &[
+            "https://raw.githubusercontent.com/protocolbuffers/protobuf/main/src/google/protobuf/descriptor.proto",
+            "https://raw.githubusercontent.com/googleapis/googleapis/master/google/type/timeofday.proto",
```

Comment:
these should be available from the protobuf crate already ...

Reply:
Can you link me? I'm seeing [timestamp](https://docs.rs/protobuf/latest/protobuf/well_known_types/timestamp/index.html) in the protobuf crate, but didn't find timeofday. Fails to compile if I remove it.

I was able to remove the descriptor.proto.

Reply:
Removed descriptor.proto. Unable to remove timeofday.proto.

Reply:
Good enough :-)